enterprise/server/filestore/filestore.go

```diff
@@ -774,7 +775,10 @@ func (g *gcsMetadataWriter) Commit() error {
 	if status.IsAlreadyExistsError(err) {
 		// This object already exists. We need to bump the
 		// custom time though.
+		log.Debugf("Write gcs blob %q (already exists), updating custom time to %d", g.blobName, g.customTime.UnixMicro())
```

Comment:
dyou wanna get the return value and log the result as well in case we're managing to swallow this in some weird way or something? or are the error catches that youre logging below sufficient?

Reply:
Done.